src/runtimes/docker/clients/DockerClientBase/DockerClientBase.ts

```diff
@@ -5,6 +5,7 @@
 
 import * as readline from 'readline';
 import { ShellQuotedString, ShellQuoting } from 'vscode';
+import { IsolationMode, getIsolationMode } from '../../../../utils/getIsolationMode';

```

Comment:
@bwateratmsft I've made the changes:
- InspectContainersItem now includes isolation property
- Moved the check isolation step outside of DockerClientBase to avoid importing external functions

Reply:
Thanks! Looks good. I'll give it a test today.

Reply:
Alright, thanks